card/mod/01_core/chunk/include.rb

```diff
@@ -122,7 +123,7 @@ def each_option attr_string
       return if attr_string.blank?
       attr_string.strip.split(';').each do |pair|
         value, key = pair.split(':').reverse
-        key ||= 'view'
+        key ||= self.class::DEFAULT_OPTION.to_s
```

Comment:
It's not supposed to be changed (except by subclasses) so a constant is probably the right choice.